iris_grib/__init__.py

```diff
@@ -1 +1,1098 @@
+# (C) British Crown Copyright 2010 - 2016, Met Office
+#
+# This file is part of Iris.
```

Comment:
Not yet, but we will need to decide on this (and whether we need a header at all).